adcircpy/fort15.py

```diff
@@ -606,10 +606,10 @@ def get_tidal_forcing(self) -> str:
                         # f'{len(self.mesh.open_boundaries)}',
                     ]
                 )
-            for index, row in self.mesh.boundaries.ocean.gdf.iterrows():
```

Comment:
I'm not sure if this change necessarily makes the fort15 format correct based on the ADCIRC documentation. This seems to mess up with the order of the two separate loops on constituents in the ADCIRC documentation.


I see the issue with the old code, but I don't think the updated code in this PR looks right either!
Maybe you meant to also de-indent the `for constituent in ...` loop?! 


https://adcirc.org/home/documentation/users-manual-v51/input-file-descriptions/model-parameter-and-periodic-boundary-condition-file-fort-15/#:~:text=end%20k%20loop-,NBFR,end%20k%20loop,-ANGINN

![image](https://github.com/noaa-ocs-modeling/adcircpy/assets/77082694/2195b92a-ce19-47c3-8d79-b0dbe69d12ba)